index.md

```diff
@@ -9,11 +9,10 @@ openEQUELLA is a digital repository that provides a single platform to house you
 
 openEQUELLA has been deployed for copyright resource collections, research materials, managing and exposing materials through websites and portals, content authoring, workflow, institutional policy and organizational resources. openEQUELLA is currently in use in a wide range of schools, universities, colleges, TAFEs, departments of education, government agencies, and corporations worldwide.
 
-With over 15 years of history, openEQUELLA is a mature solution written for the Java platform.  In its recent history, openEQUELLA was proprietary software licensed to adopters by Pearson.  EQUELLA is now available as open source software via Apereo. [Learn more about Equella and Apereo here.](https://www.apereo.org/projects/equella)
+With over 15 years of history, openEQUELLA is a mature solution written for the Java platform.  In its recent history, openEQUELLA was proprietary software licensed to adopters by Pearson. openEQUELLA is now available as open source software via Apereo. [Learn more about Equella and Apereo here.](https://www.apereo.org/programs/software/openequella)
```

Comment:
Nice pick-up on EQUELLA, but that highlighted another to me. :)

```suggestion
With over 15 years of history, openEQUELLA is a mature solution written for the Java platform.  In its recent history, openEQUELLA was proprietary software licensed to adopters by Pearson. openEQUELLA is now available as open source software via Apereo. [Learn more about openEQUELLA and Apereo here.](https://www.apereo.org/programs/software/openequella)
```

Reply:
Thanks Ian. Done.